plugin/evm/vm_test.go

```diff
@@ -1088,7 +1088,7 @@ func testStickyPreference(t *testing.T, scheme string) {
 
 	// Attempt to accept out of order
 	err = vm1BlkD.Accept(t.Context())
-	require.ErrorContains(t, err, "expected accepted block to have parent") //nolint:forbidigo // uses upstream code
+	require.ErrorContains(t, vm1BlkD.Accept(t.Context()), "expected accepted block to have parent", "unexpected error when accepting out of order block") //nolint:forbidigo // uses upstream code
```

Comment:
this looks wrong

Reply:
Oh I just in-lined the error, but left the declaration - fixed.